.env.example

```
@@ -1,2 +1,4 @@
 BOT_TOKEN='YOUR_BOT_TOKEN_HERE'
 GUILD_ID='YOUR_GUILD_ID_HERE'
+OCR_SPACE_API='YOUR_OCR_SPACE_API_KEY_HERE'
+OCR_SPACE_URL='https://api.ocr.space/parse/image'
```

```suggestion
OCR_SPACE_KEY='YOUR_OCR_SPACE_API_KEY_HERE'
OCR_SPACE_API='https://api.ocr.space/parse/image'
```
These names are still a little confusing on second look, these should be better. Of course we should change them in `env.ts` too.

---

Done